modules/ppcp-compat/webpack.config.js

```diff
@@ -0,0 +1,35 @@
+const path         = require('path');
+const isProduction = process.env.NODE_ENV === 'production';
+
+module.exports = {
+    devtool: isProduction ? 'source-map' : 'eval-source-map',
+    mode: isProduction ? 'production' : 'development',
+    target: 'web',
+    entry: {
+        'gzd-compat': path.resolve('./resources/js/gzd-compat.js'),
+    },
+    output: {
+        path: path.resolve(__dirname, 'assets/'),
+        filename: 'js/[name].js',
+    },
+    module: {
+        rules: [{
+            test: /\.js?$/,
+            exclude: /node_modules/,
+            loader: 'babel-loader',
```

Comment:
You should also copy `.babelrc`

Reply:
added 8f89226cb320c9b856500a7dcae39ce3580d5e6a